storage/blackhole/ha_blackhole.cc

```diff
@@ -40,6 +40,7 @@ using std::string;
 using std::unique_ptr;
 
 static PSI_memory_key bh_key_memory_blackhole_share;
+static bool blackhole_transactions;
```

Comment:
For clarity
```suggestion
static bool blackhole_transactions = false;
```

Reply:
Yes, perhaps it is OK as it does technically have transaction support but not enabled. However, I can also see the other side where it can be confusing.